docs/manual/layout/templates/twig/encoding.en.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Encoding / Escaping"
+description: "Encoding and Escaping information."
+aliases:
+    - /en/layout/templates/twig/encoding/
+weight: 70
+---
+
+
+For historic reasons Contao uses *input* encoding, but Twig embraces the more sane *output* encoding. You can read more about the topic 
+(and why you should favor output encoding) in this [OWASP article](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#rule-0-never-insert-untrusted-data-except-in-allowed-locations) about preventing Cross Site Scripting (XSS) attacks.
+
+
+## Why you should care
+
+The gist: You, as a template designer, have to decide how things should be output, because *you* know the context and which content you 
+trust or not. The *exact same* data can be dangerous in one context and harmless in another:
+
+With Twig we can be specific how a certain variable should be treated. Use the `|escape` or - short - `|e` 
+[filter](https://twig.symfony.com/doc/3.x/filters/escape.html) for this:
+
+```twig
+<style>
+  .box { background: {{ color|e('css') }} }
+</style>
+
+[…]
+
+<div class="box">{{ color|e('html') }}</div>
+```
+
+{{% notice note %}}Hall
+By default Twig encodes **all** variables. The chosen escaper strategy will depend on the template's file extension: your `.html.twig`
+templates will automatically get the `|e('html')` treatment, so you could omit this part in the above example.
+{{% /notice %}}
+
+You can mark a whole section of a template to be escaped or not by using the [autoescape](https://twig.symfony.com/doc/3.x/tags/autoescape.html) tag.
+
+
+## Trusted content
+
+If you want to intentionally output a variable that contains HTML, you must add the 
+[Filter](https://twig.symfony.com/doc/3.x/filters/raw.html) `|raw` to the variable. Remember that you should always use `|raw` only in 
+connection with trusted content. However, in connection with input via the TinyMCE editor in the backend, you can use the `|raw` filter in 
+this particular case.
```

Comment:
```suggestion
If you want to intentionally output something "as is" - like a variable, that contains HTML, you must add the
[Filter](https://twig.symfony.com/doc/3.x/filters/raw.html) `|raw` to the variable. Remember that you must **never** use `|raw` if you do not trust the data, otherwise this will create an XSS vulnerability.

A typical use case, where it is fine to use `|raw`, is back end content entered via the TinyMCE editor. Contao will already sanitize the user input (based on the sanitizer settings) before it passes the data to the template.
```